tests/test_cli.py

```diff
@@ -293,6 +299,30 @@ def test_correct_place_of_symlink_after_creation_of_it(self, mock_isfile):
              call(dst='/dst_folder/new_loc_add', src='/new_loc_add')],
             mock_symlink.call_args_list)
 
+    def test_delete_all_files_and_symlinks_in_destination_folder(self):
+        """ delete function should delete both file(s) and symlink(s) in destination folder. In this
+        test a temporary file and a symlink of it is created. Both of them should be deleted.
+        delete function should also delete the symlink(s) regardless of the state of existence of
+        the reference file. In this test, the symlink is existing without a reference file. The test
+        shall remove that symlink.
+        """
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            with tempfile.NamedTemporaryFile(dir=tmpdirname) as tmpfile:
+                os.symlink(src=tmpfile.name, dst=tmpfile.name + '_symlink')  # symlink creation
+                broken_symlink_path = os.path.join(tmpdirname, 'broken_symlink')
+                os.symlink(src='/tmp/blah', dst=broken_symlink_path)
```

Comment:
The code I sent you was just an example, please use a source path inside your temporary directory, so that you can be sure it's not an existing file.

Reply:
Done in the next commit